src/modules/graphics/Graphics.h

```diff
@@ -436,6 +440,7 @@ class Graphics : public Module
 	ParticleSystem *newParticleSystem(Texture *texture, int size);
 
 	Shader *newShader(const std::vector<std::string> &stagessource);
+	Shader *newComputeShader(std::string source);
```

Comment:
Should be `const std::string &source` so it doesn't make a temporary allocation for the source string every time it's called